app/retail/views.py

```diff
@@ -67,8 +67,6 @@ def get_activities(tech_stack=None, num_activities=15):
 
 def index(request):
 
-    user = request.user.profile if request.user.is_authenticated else None
```

Comment:
Looks like this user get was just accidentally leftover?

Reply:
yeah it was never used :P randomly just lying there